README.md

```diff
@@ -2,16 +2,48 @@
 
 This repository contains the source code of the Zeebe Get-Started Java client tutorial.
 
-You can find the tutorial in the [Zeebe documentation](http://docs.zeebe.io/clients/java-client/get-started).
+You can find the tutorial in the [Zeebe documentation](https://docs.camunda.io/docs/product-manuals/clients/java-client/get-started).
 
 * [Web Site](https://zeebe.io)
-* [Documentation](https://docs.zeebe.io)
+* [Documentation](https://docs.camunda.io)
 * [Issue Tracker](https://github.com/zeebe-io/zeebe/issues)
 * [Slack Channel](https://zeebe-slackin.herokuapp.com/)
 * [User Forum](https://forum.zeebe.io)
 * [Contribution Guidelines](/CONTRIBUTING.md)
 
-## Run with Maven
+The code exists in two version:
+* `ApplicationCamundaCloud`- Client connecting to cluster in Camunda Cloud
+* `ApplicationDocker` - Client connecting to a cluster running in a Docker container
```

Comment:
This reads like it connects to a cluster where that cluster is running in a Docker container.

I also don't see any files or directory structures resembling these names. I see source code to build a version locally. You can then run that version or build a docker image with the Dockerfile and a docker-compose file to start it without having to build anything locally.

Reply:
Yes, that was an intermediate version in which I had two files. I have since united the files and now removed these two lines.